src/alpineer/load_utils.py

```diff
@@ -532,11 +532,22 @@ def fov_to_ome(
     # Reorder the DataArray as OME-TIFFs require [Channel, Y, X]
     fov_xr: xr.DataArray = load_imgs_from_tree(
         data_dir=data_dir, img_sub_folder=img_sub_folder, fovs=fovs, channels=channels
-    ).transpose("fovs", "channels", "cols", "rows")
+    )
+
+    # convert to numpy for axes swapping without image rotation
+    fov_np = fov_xr.to_numpy()
+    fov_np = np.swapaxes(fov_np, 2, 3)
+    fov_np = np.swapaxes(fov_np, 1, 2)
+
+    fov_xr_reordered = xr.DataArray(
+        fov_np,
+        coords=[fov_xr.fovs, fov_xr.channels, fov_xr.rows, fov_xr.cols],
+        dims=["fovs", "channels", "rows", "cols"],
+    )
```

Comment:
Can this be done with `xr.DataArray.transpose` instead?

```
fov_xr = fov_xr.transpose("fovs", "cols", "channels", "rows")
```

Not sure if these are the exact axes to swap, but this will be a whole lot easier than extracting the numpy array and then recreating the xarray.

Reply:
That's what was done originally and it results in the image rotating 90 degrees.

Reply:
Okay wait hold on I think it was a "cols" and "rows" mix up before!! This still might work.